clients/fides-js/src/lib/i18n/i18n-utils.ts

```diff
@@ -66,11 +68,22 @@ function extractMessagesFromExperienceConfig(
       // For each translation, extract each of the translated fields
       (translation: ExperienceConfigTranslation) => {
         const locale = translation.language;
+        const localeHasOverride =
+          experienceTranslationOverrides?.override_language === locale;
```

Comment:
this has not yet been tested. We need to ensure the format of override_language matches what we have as enums on the experience translation